cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90

```diff
@@ -645,6 +646,36 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
        call abort_ice(trim(errmsg))
     endif
 
+    ! Netcdf output created by PIO
```

Comment:
Should this section be inside or outside the CESM ifdef?

Reply:
Probably outside would be better? I am not sure. We (ACCESS) compile with the CESMCOUPLED flag on currently.

Reply:
Right, and we (UWM) do not have CESMCOUPLED set so I think we'll need them outside.

Reply:
Ok - I will update

Reply:
@anton-seaice Let me test before you move it outside. I'm just now looking at the code changes. Generally in UWM we are going to be setting these options via namelist.